roles/kube_nginx_ingress_monitoring/templates/nginx_ingress_servicemonitor.yaml.j2

```diff
@@ -0,0 +1,20 @@
+apiVersion: monitoring.coreos.com/v1
+kind: ServiceMonitor
+metadata:
+  name: "kube-prometheus-nginx-ingress"
+  namespace: '{{ kube_prometheus_namespace }}'
+  labels:
+    app: nginx-ingress
+    prometheus: kube-prometheus
+spec:
+  endpoints:
+  - interval: 15s
+    port: 'metrics'
+  jobLabel: app
+  namespaceSelector:
+    matchNames:
+    - {{ nginx_ingress_namespace }}
+  selector:
+    matchLabels:
+      app: nginx-ingress
```

Comment:
Should we also filter on `release`, in case someone decides to deploy a second `nginx-ingress` in the same namespace?